doc.md

```diff
@@ -823,6 +824,14 @@ Format string of the file time shown in the info column when it matches this yea
 
 Format string of the file time shown in the info column when it doesn't match this year.
 
+## locale (string) (default `''`)
+
+An IETF BCP 47 language tag for specifying locale used when when using sort type
```

Comment:
You wrote "when" twice. Also might be good to specify an example so it is helpful for users. I think try something like this: 

```suggestion
An IETF BCP 47 language tag (e.g. `zh-CN`) for specifying the locale used when using sort type
```